src/GafferImageBindings/ImageWriterBinding.cpp

```diff
@@ -42,16 +44,48 @@
 
 #include "GafferImageBindings/ImageWriterBinding.h"
 
+using namespace std;
+using namespace boost::python;
 using namespace GafferImage;
 using namespace GafferDispatchBindings;
 
+struct DefaultColorSpaceFunctionCreator
```

Comment:
Should be inside an anonymous namespace. Also, the "Creator" part of the name seems a bit misleading, maybe just `DefaultColorSpaceFunction`?